src/main/madara-app.ts

```diff
@@ -190,7 +190,93 @@ export async function startApp(window: BrowserWindow, appId: string) {
   if (appConfig.appType === 'binary') {
     await startAppBinary(window, appConfig);
   } else if (appConfig.appType === 'docker') {
-    await startAppDocker(window, appConfig);
+    // const containerNames = appConfig.containers.map((container) =>
+    //   getContainerName(container, appConfig)
+    // );
+
+    const containerName = getContainerName(appConfig.containers[0], appConfig);
+
+    const runningContainers = await docker.listContainers();
+    // const isAppRunning = runningContainers.some((container) =>
+    //   containerNames.includes(container.Names[0].replace(/^\//, ''))
+    // );
+    const isAppRunning = runningContainers.some(
+      (container) => container.Names[0].replace(/^\//, '') === containerName
```

Comment:
what is this replace for? can we add a comment as well for how the name are before the replace?